From wfp5p@virginia.edu  Fri Aug  7 17:03:36 2009
From: Bill Pemberton <wfp5p@virginia.edu>
Date: Thu,  6 Aug 2009 15:22:01 -0400
Subject: Staging: serqt_usb2: fix qt_open parameters
To: greg@kroah.com
Message-ID: <1249586521-10722-1-git-send-email-wfp5p@virginia.edu>


The parameter list for qt_open() was from the old non usb-serial
driver.

Signed-off-by: Bill Pemberton <wfp5p@virginia.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/serqt_usb2/serqt_usb2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/serqt_usb2/serqt_usb2.c
+++ b/drivers/staging/serqt_usb2/serqt_usb2.c
@@ -874,7 +874,7 @@ static void qt_release(struct usb_serial
 }
 
 static int qt_open(struct tty_struct *tty,
-	    struct usb_serial_port *port, struct file *filp)
+		   struct usb_serial_port *port)
 {
 	struct usb_serial *serial;
 	struct quatech_port *quatech_port;
